[[ Bug 22645 ]] Add NSBluetoothAlwaysUsageDescription to iOS plist

montegoulding opened the pull request:
This patch adds the `NSBluetoothAlwaysUsageDescription` key and value to the
iOS template plist. If this key does not exist the application will crash
on iOS 13 when using bluetooth functions.

runrevmark commented on engine/rsrc/standalone-mobile-Info.plist:
```diff
@@ -72,5 +72,7 @@
 	<key>NSLocationAlwaysUsageDescription</key>
 	<string>This application requires access to Location Services always</string>
+	<key>NSBluetoothAlwaysUsageDescription</key>
+	<string>This application requires access to BlueTooth always</string>
```
`Bluetooth` not `BlueTooth`

montegoulding committed:
[[ Bug 22645 ]] Add NSBluetoothAlwaysUsageDescription to iOS plist template

This patch adds the `NSBluetoothAlwaysUsageDescription` key and value to the
iOS template plist. If this key does not exist the application will crash
on iOS 13 when using bluetooth functions.

livecodepanos commented:
@livecode-vulcan review ok 3a5585f

livecode-vulcan commented:
💙 review by @livecodepanos ok 3a5585f31b52cc960461f1d168867e5c3f4a934f

livecode-vulcan commented:
😎 test success 3a5585f31b52cc960461f1d168867e5c3f4a934f

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/751)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/160)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/160)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/160)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/160)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2277)
* try-community-universal-ios-iphoneos13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.4/builds/11)
* try-community-universal-ios-iphonesimulator13.4: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.4/builds/11)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2306)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2293)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2292)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2327)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2267)
